sdk/storage/azblob/hashing/transfer_validation_option.go

```diff
@@ -0,0 +1,14 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package hashing
+
+type StorageTransferValidationOption uint8
```

Comment:
The pattern is to use the suffix `Type` for constants. Also, can we shorten the name? How about `TransferValidationType`?

Reply:
Also, we typically just name the file `constants.go` for ones that contain all constants.

Reply:
Just move to `blob/constants.go` per other discussion.